src/views/virtualmachines/details/tabs/configuration/initialrun/components/InitRunTabCloudInit.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react';
+
+import { V1VirtualMachine, V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { CloudInitDescription } from '@kubevirt-utils/components/CloudinitDescription/CloudInitDescription';
+import { CloudinitModal } from '@kubevirt-utils/components/CloudinitModal/CloudinitModal';
+import { useModal } from '@kubevirt-utils/components/ModalProvider/ModalProvider';
+import VirtualMachineDescriptionItem from '@kubevirt-utils/components/VirtualMachineDescriptionItem/VirtualMachineDescriptionItem';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+
+type InitRunTabCloudInitProps = {
```

Comment:
Shortening 'Initial' isn't really saving any space in these component names. Consider spelling it out fully.

Reply:
sure added